### ISSUE TITLE
HTML5: Implement a branded focus state

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#3852]
Implement a branded focus state to overwrite (but not lose the idea of) the default border line added by browsers:
![focus_state](https://cloud.githubusercontent.com/assets/6312397/25448262/e1bbbee2-2a85-11e7-9595-059af829a4b0.png)

**Comment by Klauswk** [comment]
We came with the following solution:

Add to the body css this snippet:

```
*:focus {
  outline: #1b658c solid 2px !important;
  background-color: #d1e0e8 !important;
}
```

This will overwrite the focus content on all html, the result can be seen below:

![screenshot from 2017-05-16 09-26-48](https://cloud.githubusercontent.com/assets/17770979/26106051/fbe919aa-3a1a-11e7-8b79-de23a2776d55.png)

**Comment by KDSBrowne** [comment]
I'm not too sure what design might be best suited for the branded focus, however the current colors used don't meet the contrast requirements for accessibility.

http://webaim.org/resources/contrastchecker/

**Comment by antobinary** [comment]
Addressed in #4557
